vizro-core/tests/unit/vizro/models/_components/form/test_dropdown.py

```diff
@@ -188,16 +147,53 @@ def test_set_action_via_validator(self, identity_action_function):
 class TestDropdownBuild:
     """Tests model build method."""
 
-    def test_dropdown_with_all_option(self, expected_dropdown_with_all):
-        dropdown = Dropdown(options=["A", "B", "C"], id="dropdown_id", title="Title").build()
-
-        result = json.loads(json.dumps(dropdown, cls=plotly.utils.PlotlyJSONEncoder))
-        expected = json.loads(json.dumps(expected_dropdown_with_all, cls=plotly.utils.PlotlyJSONEncoder))
-        assert result == expected
-
-    def test_dropdown_without_all_option(self, expected_dropdown_without_all):
-        dropdown = Dropdown(options=["A", "B", "C"], multi=False, id="dropdown_id", title="Title").build()
-
-        result = json.loads(json.dumps(dropdown, cls=plotly.utils.PlotlyJSONEncoder))
-        expected = json.loads(json.dumps(expected_dropdown_without_all, cls=plotly.utils.PlotlyJSONEncoder))
-        assert result == expected
+    def test_dropdown_with_all_option(self):
+        dropdown = Dropdown(
+            options=["A", "B", "C"],
+            title="Title",
+            id="dropdown_id",
+        ).build()
```

Comment:
Can this be put into one line? or is it exceeding the max length?

Reply:
Let's definitely do that 😄  @nadijagraca - can you just create a separate ticket for this? No need to implement this here already.

Reply:
Personally I think I prefer it but no strong feelings.

The alternative which would solve many existing cases and keep black with its default options is we just do a one off regex to remove all trailing commas now. Then any which are better suited to being on multiple lines can have the `,` manually added back in.

Anything which requires ongoing manual work or thought by developers or reviewers would be a bad system though. So after any initial work reformatting we shouldn't need to think about it again.

Reply:
Got it - that makes sense! Let me go through some of the cases now and if I miss some, you guys just add it back in manually whenever you see something. And then we'll implement the linter now 👍